.github/workflows/ci-e2e.yml

```diff
@@ -76,15 +76,21 @@ jobs:
         permissions:
             contents: read
             id-token: write # allow issuing OIDC tokens for this workflow run
+        outputs:
+            tag: ${{ steps.build.outputs.tag }}
+            build-id: ${{ steps.build.outputs.build-id }}
+            unit-build-id: ${{ steps.build.outputs.unit-build-id }}
         steps:
             - name: Checkout
               if: needs.changes.outputs.shouldTriggerCypress == 'true'
               uses: actions/checkout@v3
-            - name: Get Docker image cached in Depot
+            - name: Build the Docker image with Depot
               if: needs.changes.outputs.shouldTriggerCypress == 'true'
               # Build the container image in preparation for the E2E tests
               uses: ./.github/actions/build-n-cache-image
+              id: build
```

Comment:
is this just the string "build"?

Reply:
Yeah this is assigning an ID to the step so that its outputs can be referenced as outputs of the job (to eventually be passed to the `depot/pull-action`):

https://github.com/PostHog/posthog/pull/18216/files#diff-3ce26ccacf5c6f81ff8b72f0d50589fc67bbb4049f8931e0fc1471968b894131R79-R82